doc/history.html

```diff
@@ -2707,6 +2707,46 @@ <h3>Development Series</h3>
               </li>
             </ul>
           </li>
+          <li>
+            <a id="v3_10_0"><b>v3_10_0</b></a> released on December 1, 2022 (<a
+              href="http://glideinwms.fnal.gov/doc.v3_10_0/index.html"
+              >Manual</a
+            >,
+            <a href="http://glideinwms.fnal.gov/doc.v3_10_0/install.html"
+              >Installation instructions</a
+            >)<br />
+            <ul>
+              <li>All the features and fixes in v3_9_6</li>
+              <li>
+                Use `SINGULARITY_DISABLE_PID_NAMESPACES` disable `--pid` in
+                Singularity/Apptainer (OSG SOFTWARE-5340, PR
```

Comment:
PR number seems to be missing or is it because this is a placeholder for when PRs are merged later as part of this release?

Reply:
These are auto-generated. The script is probably tripped by "#". Good catch. OK for now but I'll fix the script